### ISSUE TITLE
Date Picker Dialog Example incorrectly uses aria-selected on button

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-practices#1239]
https://w3c.github.io/aria-practices/examples/dialog-modal/datepicker-dialog.html uses `aria-selected` on a `<button>` element (with computed role `button`), which is invalid. The `button` role doesn't support `aria-selected`.

https://validator.nu doesn't catch this by checking the example since this is generated by script. However, validating the generated markup it reports this error:

> Error: Element button is missing one or more of the following attributes: aria-checked, role.
>
> ```
> dateCell"><button class="dateButton" type="button" tabindex="0" data-date="2019-10-31"  aria-selected="true">31</bu
> ```

Should the `aria-selected` be specified on the `<td>` (`gridcell`) instead?

**Comment by zcorpan** [comment]
@mcking65 said the button should have `aria-pressed` instead
